include/serialize/load_driver.h

```diff
@@ -9,7 +9,8 @@ namespace freetensor {
 
 Ref<Target> loadTarget(const std::string &txt);
 Ref<Device> loadDevice(const std::string &txt);
-Ref<Array> loadArray(const std::string &txt, const std::string &data);
+Ref<Array>
+loadArray(const std::pair<const std::string &, const std::string &> &);
```

Comment:
Do we really want a function receiving "a pair of references"?

BTW, `std::apply` can be used to invoke a dual-parameter function with a `std::pair`.